sequencing-server/src/lib/mustache/util/index.ts

```diff
@@ -0,0 +1,97 @@
+/* A wrapper for Handlebars, so that `sequencing-server` isn't polluted with helper registration and the like. */
+import Handlebars from "handlebars";
+import * as fs from 'fs';
+import { addTime as addTimeFull, subtractTime as subtractTimeFull, ISO8061toSTOL, STOLToISO8061, SeqNToISO8061, ISO8061toSeqN } from "./time.js";
+import { getEnv } from "../../../env.js";
+import { SequencingLanguage } from "../enums/language.js";
+
+// initialized to environment variable, but this can be changed at runtime.
+const environment = {
+    language: getEnv().SEQUENCING_LANGUAGE
+}
+
+/////////////// AERIE HELPERS ///////////////
+// wrappers for helpers
+function addTime(startTime: string, duration: string) {
+    return addTimeFull(startTime, duration, environment)
+}
+
+function subtractTime(startTime: string, duration: string) {
+    return subtractTimeFull(startTime, duration, environment)
+}
+
+// helper to flatten out array
+function flatten(array: any[]): string {
+    if (environment.language === SequencingLanguage.STOL || environment.language === SequencingLanguage.TEXT) {
+        return new Handlebars.SafeString(`[${array.join(", ")}]`).toString();
+    }
+    else {
+        return new Handlebars.SafeString(`[${array.join(" ")}]`).toString();
+    }
+}
+
+// helper to clean dates. must be manually invoked by the user, just in case. here, Text and SeqN are handled the same.
+function formatAsDate(date: string): string {
+    if (environment.language === SequencingLanguage.STOL) {
+        return ISO8061toSTOL(STOLToISO8061(date))
+    }
+    else {
+        return ISO8061toSeqN(SeqNToISO8061(date))
+    }
+}
+
+// TODO: FIGURE OUT A GOOD WAY TO ALLOW USER UPLOADS/DEFINITION IN AERIE MAIN? Ask about this.
```

Comment:
We won't address this feature in this PR - I suggest we remove this comment and make a ticket instead

Reply:
Looks great, thanks! I'm trying to decide if we should keep this code, or if we should delete it for now and add it back when we address that ticket. Thoughts?

Reply:
I think it makes sense to keep, just so it's also there in the code at the exact point of insertion. However, if that's not standard practice then we should remove it.

Reply:
All of this will be migrated into [this issue](https://github.com/NASA-AMMOS/aerie/issues/1646) and removed from the code here.